docs/source/onboarding_resources/computing/index.rst

```diff
@@ -0,0 +1,359 @@
+
+.. todo::
+
+  Add central comp link to this!   
+
+Once you have started a session, you will be able to create code, test simulation 
+results, or do quick calculations. Once you have finished coding, you’ll want to 
+follow the same steps as outlined above in the :ref:`Contributing New Documentation <contributing>` 
+section to save the information on GitHub. All of the same Git commands work on 
+the terminal used for cluster access the same way as for GitHub. 
+
+You will need to make sure that you have cloned your repository and are in the 
+appropriate working directory while logged into the cluster. Then you can add, 
+check the status, commit, and push information in a similar way. For many projects, 
+you will create a new GitHub repository that is research team specific to not 
+disturb the engineers coding workflow. 
+
+One difference in uploading to GitHub is that the cluster will require a password 
+to push information. This is **NOT** your GitHub password, but instead is a unique 
+token that you will need to create. `This website <https://techglimpse.com/git-push-github-token-based-passwordless/>`_ has information on creating a token. Many Sim Science users set their token to 
+never expire and save the token where they can reference it later. However, this 
+might compromise security in some cases, so regenerating a token periodically is 
+best practice. 
+
+.. _cluster_access_other:
+
+Other Useful Cluster Tips
+-------------------------
+
+#. If you get tired of typing long commands, one option is to make an alias. An alias is a shortcut command for commonly typed things. More information on how to do was written by the `Cost Effectiveness team <https://hub.ihme.washington.edu/display/CE/Setting+up+cluster+access>`_ 
+#. When your computer falls asleep, it will stop access to the cluster and cut off any jobs that were currently running. This can be problematic if a notebook needs to run overnight. There are a few different options to account for this including: screen, MOSH, or tmux. If you need to use these, ask a teammate. 
+
+.. _cluster_access_files:
+
+File Systems and Storage 
+------------------------
+
+The cluster can be confusing with where to store code and data. Our 
+team has created some best practices to use for data storage. 
+
+For code, please create a new directory under :code:`/ihme/code` with your 
+username. For example, this might be :code:`/ihme/code/lutzes`. 
+You should clone GitHub respositories to this location and have 
+all Jupyter notebooks and other code stored here. 
+
+For data files, there are two locations based on the size of the 
+data file. 
+
+#. For small data files, store these on GitHub in the same location as your code. A "small" file is something without thousands of rows of data. Examples might include: a list of nicknames, disease severity proportions by age/sex group, or drug efficacy data. 
```

Comment:
```suggestion
#. For small data files, store these on GitHub in the same location as your code. Examples might include: a list of nicknames, disease severity proportions by age/sex group, or drug efficacy data. The absolute maximum file size on GitHub is 100 megabytes, but be mindful of including any file over 10 megabytes, especially if there are many such files or if the file changes frequently. Too many large files can slow down the process of making new clones of the repository.
```

Maybe we shouldn't delete, but you can definitely get away with a few thousand rows depending on the columns etc!